vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Reference.java

```diff
@@ -53,6 +53,8 @@ public class Reference extends VSpaceElement implements IReference {
     private String type;
 
     private String note;
+    
+    private boolean visibility;
```

Comment:
boolean variables should be adjectives so that the methods make sense when you call them `is...` (i.e. `isVisibility` does not make much sense...)